101-container-registry/azuredeploy.parameters.json

```diff
@@ -2,14 +2,11 @@
   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "acrName": {
+    "registryName": {
       "value": "GEN-UNIQUE"
```

Comment:
NIT: registry name doesn't allow dashes.

Reply:
That's a convention for the quickstart templates.. The bot replaces a valid value.